BookingsAPI/Bookings.DAL/Commands/AddParticipantsToVideoHearingCommand.cs

```diff
@@ -56,7 +61,19 @@ public async Task Handle(AddParticipantsToVideoHearingCommand command)
             }
 
             _context.Update(hearing);
-            await _hearingService.AddParticipantToService(hearing, command.Participants);
+            
+            var participants = await _hearingService.AddParticipantToService(hearing, command.Participants);
+
+            var participantLinks = await _hearingService.CreateParticipantLinks(participants, command.LinkedParticipants);
+            
+            foreach (var participantLink in participantLinks)
+            {
+                var interpreteeLink = new LinkedParticipant(participantLink.LinkedId, 
+                    participantLink.ParticipantId, LinkedParticipantType.Interpretee);
+
+                await _context.LinkedParticipant.AddRangeAsync(participantLink, interpreteeLink);
```

Comment:
Use the `AddLink` in participant rather than adding the entity directly.

Reply:
This is actually creating the link and adding it as a row to the DB. Assigning the links to the participant is done using `AddLink` in `CreateParticipantLinks` using the private method `UpdateParticipantsWithLinks`

Reply:
At this point the links have already been added to the participant, we're simply writing them to the DB